Cargo.toml

```diff
@@ -14,12 +14,13 @@ lto = true
 default = []
 pg12 = ["pgx/pg12", "pgx-tests/pg12"]
 pg13 = ["pgx/pg13", "pgx-tests/pg13"]
+pg14 = ["pgx/pg14", "pgx-tests/pg14"]
 
 [dependencies]
 bincode = "1.3.1"
 serde = { version = "1.0", features = ["derive"] }
-pgx = {git="https://github.com/JLockerman/pgx.git", branch="guardless"}
-pgx-macros = {git="https://github.com/JLockerman/pgx.git", branch="guardless"}
+pgx = "0.2.0"
+pgx-macros = "0.2.0"
 
 [dev-dependencies]
-pgx-tests = {git="https://github.com/JLockerman/pgx.git", branch="guardless"}
\ No newline at end of file
+pgx-tests = "0.2.0"
```

Comment:
might want to up this to `0.2.1` or do a `cargo update`

Reply:
In fact, there's going to be a `0.2.2` soon...

Reply:
(they're mostly schema generator errors, so the update can wait until that PR)

Reply:
Yes, I would like to update this in a future change.